src/SFA.DAS.FAT.Web.UnitTests/Models/WhenCreatingProviderViewModel.cs

```diff
@@ -17,12 +20,15 @@ public void Then_Maps_The_Fields(Provider source)
             actual.Should().BeEquivalentTo(source, options => options
                 .Excluding(c=>c.OverallAchievementRate)
                 .Excluding(c=>c.NationalOverallAchievementRate)
-                .Excluding(c=>c.NationalOverallCohort));
+                .Excluding(c=>c.NationalOverallCohort)
+                .Excluding(c => c.DeliveryModes));
 
             actual.OverallAchievementRatePercentage.Should().Be($"{(Math.Round(source.OverallAchievementRate.Value)/100):0%}");
             actual.NationalOverallAchievementRatePercentage.Should().Be($"{(Math.Round(source.NationalOverallAchievementRate.Value)/100):0%}");
         }
 
+        // achievement rate
+
```

Comment:
What's this comment for?

Reply:
loads of tests in this file, using the comments to organise them a bit